definitions/v7/lat-team-api.yml

```diff
@@ -113,24 +107,12 @@ search:
       selector: name:contains(VOSE)
       optional: true
       filters:
-        - name: re_replace
-          args: ["^ *\\[[^\\]]*\\] *", ""] # Remove prefix tags
-        - name: re_replace
-          args: ["(?i)\\bT(\\d+)", "S$1"]
-        - name: re_replace
-          args: ["UHDRip", "BDRip"] # Fix for Radarr
         - name: append
           args: " ENGLiSH"
     title_notvose:
       selector: name:not(:contains(VOSE))
       optional: true
       filters:
-        - name: re_replace
-          args: ["^ *\\[[^\\]]*\\] *", ""] # Remove prefix tags
-        - name: re_replace
-          args: ["(?i)\\bT(\\d+)", "S$1"]
```

Comment:
this is  for parsing the results returned

this is a non-issue anymore and all results for everything on the site is now named `SXXEYY1` no longer any `TXXEYY` in any names on the site?

Reply:
There were never any TXXEYY names on the site. The SXXEYY format is always used and it's on the naming rules of the site